### ISSUE TITLE
Unit cancel movement orders when path is temporarily blocked

### ISSUE THREAD

**Issue by TAK02** [OpenRA/OpenRA#9010]
Sometimes, when the unit path is blocked by another unit, the unit will stop moving at all.

**Comment by deniz1a** [comment]
May be related: #5968

**Comment by obrakmann** [comment]
In fact a duplicate of #5968.
